static/js/buddy_list.js

```diff
@@ -270,6 +270,12 @@ export class BuddyList extends BuddyListConf {
         // Add a fudge factor.
         height += 10;
 
+        if (this.keys.length === 0) {
+            this.container = $(this.container_sel);
+            this.container.append('<div class="user-not-found"> User not found </div>');
```

Comment:
The text should support i18n so that we can add translations to it. You can take a look at the [documentation](https://zulip.readthedocs.io/en/latest/translating/internationalization.html?highlight=i18n#internationalization-for-developers) to see how to do it.